cypress/e2e/CMS/$BlogPosts/index.js

```diff
@@ -250,4 +291,89 @@ Then('the promo image is matching the earlier selected image', () => {
+})
+
+And('body was translated as {string}', (bodyTranslated) => {
+    cy.get(`div[class*='item-body'] label:contains("${bodyTranslated}")`).should('be.visible')
+})
+
+And('{string} button is displayed', (imageDisplay) => {
+    cy.get(`summary[role='button'] span:contains("${imageDisplay}")`).should('be.visible')
+})
+
+And('the following fields are displayed under {string} label', (titleText, dataTable) => {
+    cy.get(".fieldset-legend").should('include.text', titleText).and('be.visible')
+    for (const { blogTopic } of dataTable.hashes()) {
+        cy.get(`div[id*='cgov-edit-blog-topics-wrapper'] label:contains("${blogTopic}")`).should('be.visible')
+    }
+})
+
+And('Link section under related resources was translated as {string}', (TranslatedLabel) => {
+    cy.get(`span:contains("${TranslatedLabel}")`).should('be.visible')
+})
+
+And('{string} label is displayed in the page', (contentLabel) => {
+    cy.get(`h4:contains("${contentLabel}")`).should('be.visible')
+})
+
+And('Link section under recommended content is displayed as {string}', (LinkSection) => {
+    cy.get(`div[id*='featured-item-wrapper'] span:contains("${LinkSection}")`).should('be.visible')
+})
+
+And('dropdown to add link under recommended content was translated to start with {string}', (dropDownTranslated) => {
+    cy.get(`input[value^='${dropDownTranslated}']`).should('be.visible')
+})
+
+And('{string} dropdown has the following options', (PbText, dataTable) => {
+    cy.get(`div[id*='use-wrapper'] label:contains("${PbText}")`).should('be.visible')
+    for (const { options } of dataTable.hashes()) {
+        cy.get(`select[id*='edit-field-public-use'] option:contains("${options}")`).should('be.visible')
+    }
+})
+
+And('{string} label is displayed under search Engine', (DtDisplay) => {
+    cy.get(`div[id*='posted-wrapper'] h4:contains("${DtDisplay}")`).should('be.visible')
+})
+
+Given('user is navigating to the blog {string} under {string}', (blogPost, blogSeries) => {
+    cy.visit(`${frontEndBaseUrl}${blogPost}${siteSection}${blogSeries}-${randomNum}`, { retryOnStatusCodeFailure: true })
+})
+
+And(`espanol blog's posted date is today's date`, () => {
+    const months = [
+        'enero',
+        'febrero',
+        'marzo',
+        'abril',
+        'mayo',
+        'junio',
+        'julio',
+        'agosto',
+        'septiembre',
+        'octubre',
+        'noviembre',
+        'diciembre'
+    ];
+    const month = months[date.getMonth()];
+    const day = date.getDate();
+    const expectedDate = `${day} de ${month} de ${currYear}`;
+    cy.get('p>time').should('include.text', expectedDate);
+})
+
+And('user clicks on title with url spanish path {string} site section plus {string}', (spPath, purl) => {
+    cy.get(`a[href='${spPath}/${purl}-${randomNum}']`).click();
+})
+
+Given('user is navigating to the front end site with spanish path {string} site section plus {string}', (spPath, title) => {
```

Comment:
so this step is saying site section, but in reality you do not need it 
can you rephrase step in a feature file and update this one to say 
user is navigating to the front end site with spanish path {string} plus {string}